src/test/scala/uk/gov/hmrc/ui/pages/propertyLinking/PropertySearchResultPage.scala

```diff
@@ -0,0 +1,34 @@
+/*
+ * Copyright 2025 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package uk.gov.hmrc.ui.pages.propertyLinking
+
+import uk.gov.hmrc.ui.pages.BasePage
+
+object PropertySearchResultPage extends BasePage {
+
+  def searchResult(): Unit =
+    headerCheck("Search results for BH1 7EY")
```

Comment:
Add the moment search result is displaying as  "Search results for BH1 7ST". Update the header with BH1 7ST. Once the bug has been fixed, we can update the header.